test/services/WelshTextBuilderSpec.scala

```diff
@@ -203,6 +203,44 @@ class WelshTextBuilderSpec extends BaseSpec with WelshLanguage {
     }
   }
 
+  "TextBuilder expandLabels" must {
+    val labelsMap = Map(
+      "BLAH"->ScalarLabel("BLAH", List("33.5")),
```

Comment:
In this definition you are defining the English label values. You really need tom define the Welsh (i.e. ScalarLabel("BLAH", List("33.5"), List("Welsh: 33.5"). The current tests work because the labels default to English.

Reply:
added